src/client/js/create.js

```diff
@@ -409,11 +409,11 @@ async function getUserData() {
   const result = await request.json();
   console.log(result, "api result");
   userId = result.userId;
-  username = result.username;
+  username = res.username;
```

Comment:
res라고 하면 아마도 undefined가 찍히지 않을까 생각합니다. 409번째 줄에 result로 받아온 건데 그것을 넣어주는게 맞지 않을까요? 혹시 놓친거 같아서 코멘트 남깁니다.
```suggestion
  username = result.username;
```